server/graphql/common/transactions.ts

```diff
@@ -86,17 +99,37 @@ const remoteUserMeetsOneCondition = async (req, transaction, conditions): Promis
 
 /** Checks if the user can see transaction's attachments (items URLs, attached files) */
 export const canRefund = async (transaction, _, req): Promise<boolean> => {
-  if (transaction.type !== TransactionTypes.CREDIT || transaction.OrderId === null) {
+  const timeLimit = moment().subtract(30, 'd');
```

Comment:
`canRefund` will return false for host admins after 30 days too, is this what we want?